README.md

```diff
@@ -60,7 +60,14 @@ ENV["MYSQL_DATABASE"] = "test_db"
 
 ## Development
 
-After cloning run `crystal deps` or `crystal deps update`.
+After cloning run:
+
+```
+git submodule init && git submodule update
+./script/setup-test-db.sh
+```
+
+Then run `crystal deps` or `crystal deps update`.
```

Comment:
Probably a good idea to replace this with "`shards install` or `shards update`", since `shards` is a go-to package manager in Crystal ecosystem, and `crystal deps` delegates to it either way.